lzy/graph-executor/src/main/java/ai/lzy/graph/api/SchedulerApiImpl.java

```diff
@@ -26,6 +27,7 @@
 import static ai.lzy.util.grpc.GrpcUtils.newGrpcChannel;
 
 @Singleton
+@Requires(notEnv = "test-mock")
```

Comment:
you may add `@Primary` annotation on the mock scheduler-api implementation, then this `@Requires` may be ommitted